layershellev/src/events.rs

```diff
@@ -121,6 +126,7 @@ pub enum ReturnData<INFO> {
     WlBuffer(WlBuffer),
     RequestBind,
     RequestExist,
```

Comment:
I think this should be called `RequestExit` ?

Reply:
Yes, it is a typo, you can help fix it for me